trafficcloud/api.py

```diff
@@ -4,19 +4,21 @@
 import os
 import shutil
 from app_config import get_project_path, get_project_video_path, update_config_without_sections, get_config_without_sections
-import pm
 import subprocess
 import video, feat_config
 
+from pm import create_project
+
 from plotting.make_object_trajectories import main as db_make_objtraj
 
-def saveFiles(diction, *args):
-    return pm.ProjectWizard(diction).identifier
+def uploadVideo(identifier, video_filename):
```

Comment:
Isn't this file fairly superfluous at this point? Why not move all these function under the handlers that call them? Then, in the future, if we need to change them, we can edit the handlers that now have these functions. In addition, we can remove this file, and in turn, this whole trafficcloud folder.